### ISSUE TITLE
Добавление новых спутниковых карт МО

### ISSUE THREAD

**Issue by myadzel** [wladich/nakarte#1245]
Наткнулся на уникальные карты, именуются как ОФМ МО. 

https://rgis.mosreg.ru/v3/#/map?bbox=38.80369,55.88690,38.84558,55.89635

https://rgis.mosreg.ru/wmts/ofm/18/159341/81775.jpeg

**Comment by myadzel** [comment]
![Screenshot_2024-08-22-10-41-54-69_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/8e088f2e-bcb0-454a-abef-9ca700e8ebf5)

**Comment by tarwirdur** [comment]
> https://rgis.mosreg.ru/wmts/ofm/18/159341/81775.jpeg

`{"status":401,"message":"Unauthorized"}`

**Comment by myadzel** [comment]
Да, при беглом осмотре, там через заход на сайт выдают куки. Т. е. нужно фоном получить «доступ», если внедрять в сервис этот слой.
